packages/bridge/contracts/l2/L2DaiGateway.sol

```diff
@@ -164,14 +143,20 @@ contract L2DaiGateway is L2CrossDomainEnabled {
     address to,
     uint256 amount,
     bytes calldata data
-  ) external onlyL1Counterpart(l1Counterpart) returns (bytes memory) {
+  ) external override onlyL1Counterpart(l1Counterpart) {
     require(l1Token == l1Dai, "L2DaiGateway/token-not-dai");
 
```

Comment:
It might be worth adding a comment here that there is the assumption that in a valid L1 to L2 call between the gateways, we expect `finalizeInboundTransfer` not to revert. A revert on this codepath could lead to loss of user funds